packages/esm-patient-tests-app/src/test-results/filter/filter-context.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { createContext, useReducer, useEffect, useMemo } from 'react';
 import { isObject } from 'lodash-es';
-import { parseTime } from '../panel-timeline/helpers';
```

Comment:
Got rid of the panel-timeline helpers file; moved its contents to where they are actually used.